egi_notebooks_hub/egiauthenticator.py

```diff
@@ -272,5 +308,55 @@ async def pre_spawn_start(self, user, spawner):
         if not auth_state:
             # auth_state not enabled
             return
+        if self.map_users:
+            headers = {
+                "content-type": "application/json",
+                "x-auth-token": self.oneprovider_token,
+            }
+            http_client = AsyncHTTPClient()
+            user_id = auth_state.get("onedata_user")
+            user_mapping_url = (
+                f"https://{self.oneprovider_host}/api"
```

Comment:
if you have configured the var like this : oneprovider_host: "notebooks-cesnet-00.datahub.egi.eu" i think you miss here the correct port to use ( 9443), so either the port is part of the oneprovider_host var (better) or you should add it to the code

Reply:
while this should be like that, I was doing all the tests without the port and the oneprovider was replying... will do more tests

Reply:
maybe this is causing the failure when you log in?

Reply:
guess so:-)

Reply:
made it configurable